docs/documentation/customize/with-webpack.html

```diff
@@ -174,10 +228,17 @@
 <hr>
 
 {% include components/step.html
-  title="3. Create a webpack config"
+  title="3. Create a webpack config (Webpack < 3)"
```

Comment:
Small type here. Should be `(Webpack <= 3)`

Reply:
Fixed